engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -121,9 +124,16 @@
     private RenderingConfig renderingConfig = config.getRendering();
     private RenderingDebugConfig renderingDebugConfig = renderingConfig.getDebug();
 
-    public WorldRendererLwjgl(WorldProvider worldProvider, ChunkProvider chunkProvider, LocalPlayerSystem localPlayerSystem, GLBufferPool bufferPool) {
+    // TODO: work on the update/preRenderUpdate methods
+    // TODO: rendering process as constructor input and setRenderingProcess method
+    // TODO: move config-provided variables in preRenderUpdate() method?
+    // TODO: examine the potential to avoid allocation of variables such as Materials
```

Comment:
Totally fine IMHO, that's what TODOs are for! Especially if they actually get done :-)